s3parq/publish_parq.py

```diff
@@ -1,14 +1,11 @@
-import boto3
+import boto3, s3fs, re, sys, logging
```

Comment:
Are we doing this elsewhere? Im mostly just worried about consistency since these are all separate things